src/main/java/com/mazyavr/schedule/controller/TaskController.java

```diff
@@ -12,6 +14,7 @@
 
 import java.time.ZonedDateTime;
 
+@Tag(name="Таск контроллер", description="Контролер для работы с тасками")
```

Comment:
Аналогично, и таски по-русски задачи

```suggestion
@Tag(name="Котроллер задач")
```